### DISCUSSION TITLE
89

### DISCUSSION THREAD

**Discussion by giscus** [sukvvon/blog-comments#26] [Comments]
# 89

https://github.com/prgrms-fe-devcourse/FEDC4_Campers_Jaeho 9월은 팀 프로젝트를 하는 달이었다. 인생을 살면서 첫 프로젝트이기도 하고, 팀장도 처음으로 맡아서 하게 되었다. 개발 더 나아가 인생을 살면서 잊지 못하는 순간 중 하나라는 생각이 든다. 다사다난했던 한 달이었기 때문이다. 기술적인 측면보다는 경험적인 측면에서 회고를 해보려고 한다. 개발에 대한 회고 좋았던 점 초기 설정 (husky, prettier, eslint, lint-staged, commitlint, github) axios (auth) auth page 및 회원가입 컴포넌트 공통화, alert-dialog 창, tabs 창 context API (userInfo) 크게 위와 같이 나누어..

https://sukvvon.tistory.com/89

**Comment by hyeonjun-L** [comment]
한달 고생 많았어 😀
